.changeset/dull-colts-fetch.md

```diff
@@ -0,0 +1,108 @@
+---
+'sku': major
+---
+
+Remove Storybook-related features from `sku`
+
+**BREAKING CHANGE**
+
+All Storybook-related features have been removed from `sku`. Sku no longer installs Storybook dependencies, configures Storybook, provides Storybook CLI commands, or re-exports Storybook APIs. If you are using Storybook, you will need to install Storybook dependencies yourself. See [`sku`'s Storybook documentation][storybook docs] for more information.
+
+[storybook docs]: https://seek-oss.github.io/sku/#/./docs/storybook
+
+**MIGRATION GUIDE**:
+
+#### Commit your Storybook configuration
+
+Run `sku configure` to update your ignore files:
+
+```sh
+pnpm exec sku configure
+```
+
+This will allow you to check in the `.storybook/main.ts` file to your repository. Ensure you include both of these changes **in the same commit**.
+
+#### Install Storybook dependencies
+
+`sku` was previously installing Storybook v7 dependencies for you. The first command below will install the latest (v8) Storybook dependencies. If you wish to stay on Storybook v7 for the time being, ensure you specify the appropriate versions of these dependencies in your app's `package.json`.
+
+**NOTE**: Consumers that still depend on the deprecated `storiesOf` API will need to stay on Storybook v7 until they migrate away from this API.
+
+```sh
+pnpm install -D storybook @storybook/react @storybook/react-webpack5
+
+# (Optional) This addon is required for Storybook >=8.0.0
+pnpm install -D @storybook/addon-webpack5-compiler-babel
+```
+
+#### Configure Storybook
+
+Sku no longer configures your `.storybook/main.ts` file for you. You will now need to configure it yourself. See [`sku`'s Storybook documentation][storybook docs] for more information.
+
+`sku` now provides the `babel` and `webpackFinal` configuration objects under the `sku/config/storybook` entrypoint. These objects are best utilized within in order to  Here's an example of a minimal, `sku`-compatible Storybook configuration:
```

Comment:
```suggestion
`sku` now provides the `babel` and `webpackFinal` configuration objects under the `sku/config/storybook` entrypoint. These objects are best utilized within in order to  

Here's an example of a minimal, `sku`-compatible Storybook configuration:
```

Reply:
"These objects are best utilized within in order to" think this sentence got cut off too? Also utilised?